app/views/projects/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1><%= @projects.name %></h1>
+<p>Material: <%= @projects.material %></p>
+<p>Challenge Theme: <%= @projects.challenge.theme %></p>
+<p> Number of Contestants: <%= @contestants.count %></p>
+<p> Average Contestant Experience: <%= @projects.average_age %> years</p> 
+
+
+<%= form_with url: "/projects/#{@projects.id}", method: :post, local: true do |form| %>
+  <%= form.label :name %>
```

Comment:
This is close - your form and paths are set up well, but I think the story is asking you to enter the ID of an existing contestant instead of a name. Try to work through this again for good practice.